README.md

```diff
@@ -1145,12 +1147,12 @@ The `web-vitals` code has been tested and will run without error in all major br
 
 Browser support for each function is as follows:
 
-- `onCLS()`: Chromium,
-- `onFCP()`: Chromium, Firefox, Safari
-- `onFID()`: Chromium, Firefox, Safari, Internet Explorer (with the [polyfill](#how-to-use-the-polyfill))
+- `onCLS()`: Chromium
+- `onFCP()`: Chromium, Firefox, Safari 14.1+
```

Comment:
Separate to this PR, but is it worth noting somewhere in the docs (either here or elsewhere) that FCP on Safari is quite different: https://blog.webpagetest.org/posts/why-first-contentful-paint-doesnt-work-as-a-cross-browser-metric/

Happy to open a separate issue for this.

Reply:
We should probably discuss whether it's worth pointing this out in this library documentation or in our metric documentation on web.dev. Personally, I think I'm leaning toward the latter.

Reply:
Agree it should be in docs, but given this library specifically states it can be used for tracking across these browsers a think it should be linked from here. The complication with that is our docs are per metric so don’t have one place to link. But to be honest FCP is the one that I’m aware of that is the most different.

Reply:
Discussed offline. Might be an upcoming post that would be good place to link to. Happy to leave for now.